app/jobs/fraud_rejection_daily_job.rb

```diff
@@ -0,0 +1,19 @@
+class FraudRejectionDailyJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    profiles_eligible_for_fraud_rejection.find_each do |profile|
+      analytics.automatic_fraud_rejection(verified_at: profile.verified_at)
+      profile.reject_for_fraud(notify_user: false)
+    end
+  end
+
+  private
+
+  def profiles_eligible_for_fraud_rejection
+    Profile.where(
+      fraud_review_pending: true,
+      verified_at: ..30.days.ago,
+    )
```

Comment:
we don't have an index on `verified_at` , can we add one? otherwise this will be a table scan

Reply:
or on `fraud_review_pending` just something so that we don't scan the whole table

Reply:
Ok I'll place it on fraud_review_pending  we are looking to replace the boolean with a timestamp further down the line.